### ISSUE TITLE
Move documentation file to Github Wiki

### ISSUE THREAD

**Issue by abo3skr2019** [abo3skr2019/Student-Club-Management-System#13]
Currently there is a mix of Documentation files either in root or the documentation directory this is a request to move them to the documentation directory to allow first time contributors to find what they need faster

**Comment by Fkhrayef** [comment]
I suggest moving them to wiki. It would be easier to access and modify.

**Comment by abo3skr2019** [comment]
Agreed We should move the documentation to the wiki here is a on [guide](https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages) how to do that
